src/cache.rs

```diff
@@ -323,26 +347,58 @@ pub struct Caches {
 
 impl Caches {
     pub fn from_config(config: &Config) -> io::Result<Self> {
+        let tmp_dir = config.cache_dir("tmp");
+        if let Some(ref tmp) = tmp_dir {
+            if tmp.exists() {
```

Comment:
If the cleanup job starts, won't this get executed and the tmpdir wiped even though symbolicator is still working?

Reply:
ugh, good catch!